README.md

```diff
@@ -1,13 +1,16 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This weeks project was to create an API using Express with the help of an already hard-coded set of data, stored as a JSON file in your project. The API should have at least a couple of RESTful endpoints which return either an array of data, or a single item.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+From the beginning this weeks project looked a bit easyer than previous weeks where we worked with react and redux. For me the easy part this week was to understand how to write the code. But I got lost in the the difference between endpoints, routes, params and query and when to use what. And also how to write the paths correct both in the code but also how to write them correct in postman or the browser. This sometimes resulted in me believeing that there where something wrong with my code and I unnecessarily rewrote it...which has been really annoying.
+
+I can't say that I yet fully understand it...
+
+If I hade more time it would been fun to use my API to also do a frontend.
+Or try to figure out why my error message dosen't work when searching for book over or below 500 pages.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://small-express-books-api.herokuapp.com (and... /books ...for example)
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓